components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/TrustedApp.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.common.model;
+
+import org.apache.axis2.databinding.annotation.IgnoreNullElement;
+
+import java.io.Serializable;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * Platform specific trusted app configuration of an application.
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "TrustedApp")
+public class TrustedApp implements Serializable {
+
+    private static final long serialVersionUID = -753653473009612027L;
+    private static final String PLATFORM_TYPE = "PlatformType";
+    private static final String APP_IDENTIFIER = "AppIdentifier";
+    private static final String THUMBPRINTS = "Thumbprints";
+    private static final String IS_FIDO_TRUSTED = "IsFidoTrusted";
+
+    @IgnoreNullElement
+    @XmlElement(name = PLATFORM_TYPE)
+    private String platformType;
+
+    @IgnoreNullElement
+    @XmlElement(name = APP_IDENTIFIER)
+    private String appIdentifier;
+
+    @IgnoreNullElement
+    @XmlElement(name = THUMBPRINTS)
+    private String[] thumbprints;
+
+    @IgnoreNullElement
+    @XmlElement(name = IS_FIDO_TRUSTED)
+    private boolean isFIDOTrusted;
```

Comment:
Shall we add a code comment mentioning whats meant by "FIDOTrusted" as this is not self explanatory.